Tests/TezosKit/Helpers/FakeURLSession.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright Keefer Taylor, 2019.
+
+import Foundation
+
+/// A fake URLSession that will return data tasks which will call completion handlers with the given parameters.
+public class FakeURLSession: URLSession {
```

Comment:
Why not use a pod just for the tests that provides fake/mock'ed urlsessions rather than role your own. Even if its GTM